solutions/usaco-989.mdx

```diff
@@ -51,4 +51,51 @@ class Main {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	freopen("race.in", "r", stdin);
+	freopen("race.out", "w", stdout);
```

Comment:
this is optional, but i'd like it if you could use `std::ifstream` and `std::ofstream`
completely up to you, it wouldn't be fair to mandate that